lib/index.js

```diff
@@ -101,3 +101,6 @@ function resolveCookies(ctx, opts) {
     return ctx.cookies.get(opts.cookie);
   }
 }
+
+// Export JWT as a convenience
+module.exports.jwt = require('jsonwebtoken');
```

Comment:
The purpose of the convenience function is to provide the matching JWT version for signing tokens. Otherwise an extra library is required on the user end, and koa-jwt doesn't provide all of the methods necessary to use it.